build/ical.js

```diff
@@ -886,7 +894,9 @@ ICAL.design = (function() {
 
   // When adding a value here, be sure to add it to the parameter types!
   var vcardValues = ICAL.helpers.extend(commonValues, {
-
+    component: textType(FROM_ICAL_NEWLINE, TO_ICAL_NEWLINE, "text"),
```

Comment:
So this is always the case when it is a `structuredValue`, right? Can you just make it depend on `structuredValue` being set, and then not escaping the character defined by `structuredValue`?